src/views/Question.vue

```diff
@@ -131,16 +132,23 @@ watch:{
   max-width: 960px;
   padding: 0 36px;
   margin: 0 auto;
-  margin-top: 218px;
 }
 
 .question-wrapper {
-  background-image: url(../assets/other-3.png);
+  background-color: #DFE3E8; 
+  
+  padding-bottom: 200px;
+}
+
+.bg-img {
   background-repeat: no-repeat;
   background-position: right top;
-  background-color: #DFE3E8; 
   overflow: hidden;
-  padding-bottom: 200px;
+  -moz-transform: scaleX(-1);
+  -o-transform: scaleX(-1);
+  transform: scaleX(-1);
+  filter: FlipH;
+  -ms-filter: "FlipH";
```

Comment:
I think it would be best to flip the image itself (with photoshop or any graphic editor) instead of manipulating it with CSS here 😃

Reply:
Ok cool, I didn't know if we were allowed to modify the original image so that's why I used CSS